tests/test_simple_graph.py

```diff
@@ -1,71 +1,74 @@
 # _*_ Coding: utf-8 _*_
 
 import pytest
-from src.simple_graph import Node
-
-TEST_NODE_1 = Node(1)
-TEST_NODE_2 = Node(2)
 
 
 def test_sgraph_nodes():
     from src.simple_graph import Graph
     test_graph = Graph()
-    test_graph.add_node(2) 
+    test_graph.add_node(2)
     assert test_graph.nodes() == [2]
 
 
+def test_sgraph_nodes2():
+    from src.simple_graph import Graph
+    test_graph = Graph()
+    test_graph.add_edge(5, 6)
+    assert test_graph.nodes() == [5, 6]
+
+
 def test_sgraph_edges():
     from src.simple_graph import Graph
     test_graph = Graph()
-    test_graph.add_node(2) 
-    test_graph.add_node(3)
-    test_graph.add_node(4)
-    test_graph.add_edge(2,3)
-    test_graph.add_edge(2,4)
-    assert test_graph.edges() == [(2,3), (2,4)]
+    test_graph.add_edge(2, 3)
+    test_graph.add_edge(2, 4)
+    assert test_graph.edges() == [(2, 3), (2, 4)]
 
 
 def test_sgraph_add_node():
-    #expect new entry in dict
     from src.simple_graph import Graph
     test_graph = Graph()
-    test_graph.add_node(2) 
+    test_graph.add_node(2)
     assert test_graph.dict[2] == []
-    
+
 
 def test_sgraph_add_edge():
     from src.simple_graph import Graph
     test_graph = Graph()
-    test_graph.add_node(2) 
-    test_graph.add_node(3)
-    test_graph.add_edge(2,3)
+    test_graph.add_edge(2, 3)
     assert test_graph.dict[2] == [3]
 
+
 def test_sgraph_add_edge_add_n2():
     from src.simple_graph import Graph
     test_graph = Graph()
-    test_graph.add_node(2) 
-    test_graph.add_edge(2,3)
+    test_graph.add_node(2)
+    test_graph.add_edge(2, 3)
     assert test_graph.dict[2] == [3]
 
+
 def test_sgraph_add_edge_added_key():
     from src.simple_graph import Graph
     test_graph = Graph()
-    test_graph.add_node(2) 
-    test_graph.add_edge(2,3)
+    test_graph.add_node(2)
+    test_graph.add_edge(2, 3)
     assert 3 in test_graph.nodes()
```

Comment:
You haven't checked if the edge was actually added, just that the node was added.